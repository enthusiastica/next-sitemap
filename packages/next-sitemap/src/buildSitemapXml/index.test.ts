import { buildSitemapXml } from '.'

describe('generateSitemap', () => {
  test('buildSitemapXml', () => {
    expect(
      buildSitemapXml(
        {
          siteUrl: 'https://example.com',
          priority: 0.7,
          changefreq: 'daily',
          path: 'sitemap'
        },
        ['/', '/another', '/example']
      )
    ).toMatchSnapshot()
  })
})
