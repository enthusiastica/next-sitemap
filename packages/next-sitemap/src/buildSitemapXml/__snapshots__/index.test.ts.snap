// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateSitemap buildSitemapXml 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
<urlset xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\" xmlns:news=\\"http://www.google.com/schemas/sitemap-news/0.9\\" xmlns:xhtml=\\"http://www.w3.org/1999/xhtml\\" xmlns:mobile=\\"http://www.google.com/schemas/sitemap-mobile/1.0\\" xmlns:image=\\"http://www.google.com/schemas/sitemap-image/1.1\\" xmlns:video=\\"http://www.google.com/schemas/sitemap-video/1.1\\">
	<url>
		<loc>/</loc>
		<changefreq>daily</changefreq>
		<priority>0.7</priority>
	</url>
	<url>
		<loc>/another</loc>
		<changefreq>daily</changefreq>
		<priority>0.7</priority>
	</url>
	<url>
		<loc>/example</loc>
		<changefreq>daily</changefreq>
		<priority>0.7</priority>
	</url>
</urlset>"
`;
